frontend/src/pages/learn/index.tsx

```diff
@@ -83,30 +94,38 @@ const Learn: FC = () => {
       <Modal
```

Comment:
Dialog component should be used instead of Modal.

https://mui.com/material-ui/react-modal/
> If you are creating a modal dialog, you probably want to use the [Dialog](https://mui.com/material-ui/react-dialog/) component rather than directly using Modal.

This can be done in another PR.

Reply:
No problem